dio/lib/src/transformers/background_transformer/compute.dart

```diff
@@ -0,0 +1,134 @@
+/// This file corresponds to Flutter's
```

Comment:
Can we raise the level to `src/compute/*`?

Reply:
Sure